### ISSUE TITLE
[Request] hot fluids whitelist

### ISSUE THREAD

**Issue by Vampsku11** [KnightMiner/Ceramics#46]
Originally a request for a creosote clay bucket but now I realize this is available but disabled when hotFluids=false. Is it possible to whitelist specific fluids from being ignored by the hotFluids option?

**Comment by KnightMiner** [comment]
When I get around to it, I plan to make the temperature of what makes a fluid "hot" configurable. I do not think adding a whitelist makes much sense as the idea is too hot breaks, it sounds more like the mod added too high of a temperature for creosote oil (Immersive Engineering oil is fully reusable in clay buckets).

**Comment by Vampsku11** [comment]
I have hot fluids pickup disabled instead of just breaking the bucket when emptied. Other mods seem to have creosote at different temperatures.
I did use CraftTweaker to set the temperature of creosote below 450 degrees, but then I had to replace Tinkers Construct melting recipes so certain things could be melted at the lower temperature. I also raised the temperature of some liquids over 450 so they couldn't be bucketed.
It works to achieve my goal, but it feels kind of messy as the temperatures of liquids have to be a little inconsistent with each other.
